cluster/charts/crossplane-controllers/templates/deployment.yaml

```diff
@@ -21,6 +21,9 @@ spec:
         app: {{ template "name" . }}
         release: {{ .Release.Name }}
     spec:
+      {{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName }}
```

Comment:
should we be consistent with the ` | quote` usage? this core crossplane deployment doesn't have it but the stack manager deployment does.

Reply:
Ah, I missed that. Done.